docs/sdk/server-side-sdks/java-local/java-local-gettingstarted.md

```diff
@@ -60,8 +61,7 @@ public class MyClass {
         .disableCustomEventLogging(false)
         .build();
     
-    public MyClass() {
-        dvcLocalClient = new DVCLocalClient("<DVC_SERVER_SDK_KEY>", dvcLocalOptions);
+        client = new DVCLocalClient("<DVC_SERVER_SDK_KEY>", options);
```

Comment:
We should set a best practice of sourcing this from an env var even in our docs.